test/functional/currency/currency-formatter.js

```diff
@@ -19,7 +19,8 @@ define([
 
 var accounting = { style: "accounting" },
 	code = { style: "code" },
-	name = { style: "name" },
+    name = { style: "name" },
+	narrow = { symbolForm: "symbol-alt-narrow" },
```

Comment:
Let's simplify the API option. Instead of the above, let's use `{ form: "narrow" }` and have the formatter to include the `symbol-alt-` part when necessary.